df.refs.xml

```diff
@@ -355,7 +355,7 @@
         <enum-item name='JOB'>
             <item-attr name='union_field' value='job'/>
         </enum-item>
-        <enum-item name='BUILDING_PARTY'/>
+        (enum-item name='BUILDING_PARTY'/) removed in v50
```

Comment:
If they've been removed from the game, then remove them from the XML - there's no point in leaving them commented-out.